### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by mkljczk**
I’m going to start by translating the *Using Mastodon* section.

Preview: <a href="http://mastodocs.mkljczk.pl/pl/" target="_blank">here</a>

Progress:
- [x] What is Mastodon?
- [x] Signing up for an account
- [x] Setting up your profile
- [x] Posting toots
- [x] Using the network features
- [x] Dealing with unwanted content
- [x] Promoting yourself and others
- [x] Set your preferences
- [x] More settings
- [x] Using Mastodon externally
- [x] Moving or leaving accounts
- [x] Running your own server

- [x] Review from another Polish-speaking person

Optional?:
~~- [ ] Localized screenshots~~
~~- [ ] Translating the entire articles mentioned on the *What is Mastodon?* page~~

### CODE REVIEW

File: content/pl/user/posting.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+| Bezpośrednio | Nie | Zalogowani i wspomnieni | W aplikacji lub zalogowani | Nie |
+
+Wpisy mogą być opublikowane z czterema poziomami prywatności:
+
+### Publiczny {#public}
+
+Domyślna opcja.
+
+* Każdy może zobaczyć Twój wpis z poziomu linku bezpośredniego bez logowania się.
+* Twój wpis pojawi się w aplikacji na publicznej osi czasu.
```
Comment (hexandcube) [inline content/pl/user/posting.md:87]:
```suggestion
* Twój wpis pojawi się w aplikacji na publicznych osiach czasu.
```

File: content/pl/user/posting.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+* Twoi obserwujący zobaczą ten wpis w swoich osiach czasu, a wszyscy oznaczeni zostaną o nim powiadomieni.
+* Twój wpis może zostać udostępniony przez innych użytkowników.
+
+### Niewidoczny {#unlisted}
+
+Tak jak przy publicznym, ale z następującą różnicą:
+
+* Twój wpis nie pojawi się w publicznych osiach czasu na Mastodonie.
+
+### Tylko dla śledzących {#private}
```
Comment (hexandcube) [inline content/pl/user/posting.md:97]:
```suggestion
### Tylko dla obserwujących {#private}
```

File: content/pl/user/posting.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+Bardziej ograniczona opcja doręczania.
+
+* Zobaczenie Twojego wpisu przez odnośnik bezpośredni wymaga zalogowania na tym samym serwerze, na którym ta osoba obserwuje Cię lub została wspomniana.
+* Twój wpis nie wyświetla się w aplikacji, za wyjątkiem obserwowanych i wspomnianych osób.
```
Comment (hexandcube) [inline content/pl/user/posting.md:102]:
```suggestion
* Twój wpis nie wyświetla się w aplikacji nikomu, za wyjątkiem obserwujących i wspomnianych osób.
```

File: content/pl/user/posting.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+{{< hint style="danger" >}}
+Pamiętaj, że prywatność wpisów na Mastodonie dotyczy poszczególnych wpisów, a nie całego konta, więc **nie istnieje sposób na uczynienie starszych wpisów prywatnymi.**
+{{< /hint >}}
+
+### Bezpośrednio {#direct}
+
+Wyślij wpis wyłącznie wspomnianym użytkownikom.
+
+* Zobaczenie Twojego wpisu przez odnośnik bezpośredni jest możliwe na serwerze, na którym znajduje się osoba, na której ktoś został wspomniany.
+* Twój wpis nie pokaże się w aplikacji, jeżeli nie zostałeś(-aś) wspomniany(-a).
```
Comment (hexandcube) [inline content/pl/user/posting.md:119]:
```suggestion
* Twój wpis nie pokaże się w aplikacji nikomu, poza osobami wspomnianymi.
```

File: content/pl/user/preferences.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+{{< figure src="/assets/image%20%286%29.png" caption="Przykład miniaturki z blurhashem" >}}
+
+Wpisy z ostrzeżeniami o zawartości są domyślnie zwijane, ale możesz wybrać aby zawsze wyświetlany był pełny wpis.
+
+## Kontrolowanie powiadomień {#notifications}
+
+### Wysyłanie e-maili {#email}
+
+Możesz wybrać, aby dostawać powiadomienia e-mail według typów powiadomień, które otrzymujesz na Mastodonie. Możesz włączyć maile dla poszczególnych rodzajów powiadomień:
+You can choose to receive email notifications according to the type of notification you receive within Mastodon. The following notification types are available to enable:
```
Comment (hexandcube) [inline content/pl/user/preferences.md:53]:
```suggestion

```

File: content/pl/user/preferences.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+* Obserwacje
+* Prośby o możliwość obserwacji
+* Udostępnienia
+* Ulubione
+* Wspomnienia
+
+Możesz też włączyć maile z przeglądami, które zawierają przegląd powiadomień otrzymanych podczas dłuższej nieaktywności.
+
+### Ukrywanie części powiadomień {#hide-notifications}
```
Comment (hexandcube) [inline content/pl/user/preferences.md:63]:
```suggestion
### Ukrywanie niektórych powiadomień {#hide-notifications}
```

File: content/pl/user/profile.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
+---
+title: Dostosuj swój profil
```
Comment (hexandcube) [inline content/pl/user/profile.md:2]:
```suggestion
title: Dostosowywanie swojego profilu
```

File: content/pl/user/discoverability.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+## Katalog profilów {#directory}
+
+{{< figure src="/assets/image%20%2831%29.png" caption="Katalog profilów na mastodon.social" >}}
+
+Katalog profilów pokazuje wszystkie konta, które zgodziły się być pokazywane w katalogu i może służyć szybkiemu wyszukiwaniu profilów, którymi możesz się zainteresować.
+
+Katalog profilów może być posortowany według ostatniej aktywności \(ostatnio opublikowane wpisy\) lub od najnowszych użytkowników \(ostatnio założone konto\). Katalog może wyświetlać tylko lokalne konta lub wszystkie znane przez serwer.
+
+Profile są przedstawiane jako karty zawierające nazwę wyświetlaną użytkownika, adres, biografię i kilka podstawowych statystyk jak liczba opublikowanych wpisów, liczba obserwujących i kiedy opublikowali ostatni wpis.
```
Comment (hexandcube) [inline content/pl/user/discoverability.md:38]:
```suggestion
Profile są przedstawiane jako karty zawierające nazwę wyświetlaną użytkownika, adres, biogrami kilka podstawowych statystyk jak liczba opublikowanych wpisów, liczba obserwujących i kiedy opublikowali ostatni wpis.
```

File: content/pl/user/profile.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+* Nowi obserwujący nie będą automatycznie przyjmowani, będziesz musiał(a) ręcznie zatwierdzać ich.
+* Ikona kłódki będzie widoczna dla innych, aby wiedzieli, że obserwacja nie zostanie automatycznie przyjęta.
+
+### Konto bota {#bot}
+
+Włączenie tej flagi doda ikonkę bota do Twojego profilu. Dzięki temu inni dowiedzą się, że Twój profil może dokonywać zautomatyzowanych działań i może nie być nadzorowany przez człowieka. Inne oprogramowanie może traktować profile botów w inny sposób, ale obecnie Mastodon jedynie wyróżnia boty wizualnie.
+
+### Katalog profilów {#discoverable}
+
+Udział w katalogu profilów sprawi, że Twój profil będzie mógł zostać odnaleziony przez tę funkcję, pozwalającą na przeglądanie profilów.
```
Comment (hexandcube) [inline content/pl/user/profile.md:51]:
```suggestion
Udział w katalogu profilów sprawi, że Twój profil będzie mógł zostać odnaleziony przez funkcję, pozwalającą na przeglądanie profilów.
```

File: content/pl/user/profile.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+## Metadane profilu {#fields}
+
+Metadane profilu to możliwość dodania do profilu dodatkowych, przejrzystych informacji. Masz cztery wiersze, w których możesz wpisać podpis i zawartość pola. Na przykład:
+
+| Podpis | Zawartość |
```
Comment (hexandcube) [inline content/pl/user/profile.md:57]:
```suggestion
| Etykieta | Zawartość |
```

File: content/pl/user/profile.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+Włączenie tej flagi doda ikonkę bota do Twojego profilu. Dzięki temu inni dowiedzą się, że Twój profil może dokonywać zautomatyzowanych działań i może nie być nadzorowany przez człowieka. Inne oprogramowanie może traktować profile botów w inny sposób, ale obecnie Mastodon jedynie wyróżnia boty wizualnie.
+
+### Katalog profilów {#discoverable}
+
+Udział w katalogu profilów sprawi, że Twój profil będzie mógł zostać odnaleziony przez tę funkcję, pozwalającą na przeglądanie profilów.
+
+## Metadane profilu {#fields}
+
+Metadane profilu to możliwość dodania do profilu dodatkowych, przejrzystych informacji. Masz cztery wiersze, w których możesz wpisać podpis i zawartość pola. Na przykład:
```
Comment (hexandcube) [inline content/pl/user/profile.md:55]:
```suggestion
Metadane profilu to możliwość dodania do profilu dodatkowych, przejrzystych informacji. Masz cztery wiersze, w których możesz wpisać etykietę i zawartość pola. Na przykład:
```

File: content/pl/user/profile.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+| Wiek | 25 |
+| Kraj | Niemcy |
+| Zaimki | on/jego |
+| Strona internetowa | https://example.com |
+
+To, co tam wpiszesz, zależy wyłącznie od Ciebie. Zawartość może zawierać wspomnienia, hashtagi, niestandardowe emoji i odnośniki.
+
+### Weryfikacja odnośników {#verification}
+
+Weryfikacja z użyciem dokumentów potwierdzających tożsamość i niebieskie znaczki nie są możliwe bez centralnego zarządzania. Mastodon może krzyżowo dowiązywać odnośniki z profilem, aby udowodnić, że jesteś ich rzeczywistym właścicielem. Jeżeli jeden z tych to link do Twojej osobistej strony internetowej, która budzi zaufanie, może to służyć jako prawie najlepsza metoda weryfikacji tożsamości.
```
Comment (hexandcube) [inline content/pl/user/profile.md:68]:
```suggestion
Weryfikacja z użyciem dokumentów potwierdzających tożsamość i niebieskie znaczki nie są możliwe bez centralnego zarządzania. Mastodon może krzyżowo powiązywać odnośniki z profilem, aby udowodnić, że jesteś ich rzeczywistym właścicielem. Jeżeli jeden z nich to link do Twojej osobistej strony internetowej, która budzi zaufanie, może to służyć jako prawie najlepsza metoda weryfikacji tożsamości.
```

File: content/pl/user/profile.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+
+Jeżeli umieścisz odnośnik w metadanych swojego profilu, Mastodon sprawdzi, czy zawierra on odnośnik do Twojego profilu na Mastodonie. Jeżeli tak, otrzymasz znak weryfikacji przy tym odnośniku, ponieważ jesteś jego potwierdzonym właścicielem.
```
Comment (hexandcube) [inline content/pl/user/profile.md:70]:
```suggestion
Jeżeli umieścisz odnośnik w metadanych swojego profilu, Mastodon sprawdzi, czy powiązana strona zawiera odnośnik do Twojego profilu na Mastodonie. Jeżeli tak, otrzymasz znak weryfikacji przy tym odnośniku, ponieważ jesteś jego potwierdzonym właścicielem.
```

File: content/pl/user/run-your-own.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
+---
+title: Prowadź swój własny serwer
```
Comment (hexandcube) [inline content/pl/user/run-your-own.md:2]:
```suggestion
title: Prowadzenie własnego serwera
```

File: content/pl/user/run-your-own.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+
+Będziesz potrzebować:
+
+- **Domeny**. W ten sposób inni będą korzystać z Twojego serwera, jest to część tożsamości Twoich użytkowników w pozostałej części sieci.
+
+  **Jak uzyskać**: Namecheap, Gandi, dowolny z licznych rejestratorów domen. Wiąże się to z coroczną opłatą zależną od wybranej domeny.
+- Serwera **VPS**. To coś, na czym będzie uruchamiany serwer Mastodona, który będzie stale połączony z Internetem.
+
+  **Jak uzyskać**: DigitalOcean, Hetzner, Exoscale, Scaleway, dowolny z licznych dostawców hostingu. Wiąże się to z miesięcznym lub rocznym kosztem zależnym od specyfikacji sprzętu.
+- **Dostawca e-mail**. Mastodon musi wysyłać wiadomości z odnośnikami potwierdzającymi i różne powiadomienia przez e-mail, a choć hostowanie własnego serwera SMTP jest możliwe, może to być bardziej skomplikowane niż z korzystanie usługi podmiotu trzeciego.
```
Comment (hexandcube) [inline content/pl/user/run-your-own.md:30]:
```suggestion
- **Dostawcy e-mail**. Mastodon musi wysyłać wiadomości z odnośnikami potwierdzającymi i różne powiadomienia przez e-mail, a choć hostowanie własnego serwera SMTP jest możliwe, może to być bardziej skomplikowane niż z korzystanie usługi podmiotu trzeciego.
```

File: content/pl/user/run-your-own.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+
+{{< caption-link url="https://masto.host" caption="Masto.host" >}}
+
+{{< caption-link url="https://hostdon.jp" caption="Hostdon" >}}
+
+{{< caption-link url="https://app.spacebear.ee/mastodon" caption="Spacebear" >}}
+
+{{< caption-link url="https://nablahost.com/services/activitypub-hosting/" caption="Nablahost" >}}
+
+Zarządzanie rozwiązania hostingu są świetnym rozwiązaniem, jeżeli nie masz doświadczenia lub chęci samodzielnie instalować i zajmować się oprogramowaniem. Jednak bycie odpowiedzialnym za wszystkie elementy na własnym sprzęcie daje większą kontrolę nad skalowalnością, wydajnością i dostosowywaniem.
```
Comment (hexandcube) [inline content/pl/user/run-your-own.md:47]:
```suggestion
Zarządzane hostingi są świetnym rozwiązaniem, jeżeli nie masz doświadczenia lub chęci samodzielnie instalować i zajmować się oprogramowaniem. Jednak bycie odpowiedzialnym za wszystkie elementy na własnym sprzęcie daje większą kontrolę nad skalowalnością, wydajnością i dostosowywaniem.
```

File: content/pl/user/run-your-own.md [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+
+Oferujemy **obraz instalacji jednym kliknięciem dla DigitalOcean**, który możesz umieścić na wybranym droplecie DigitalOcean, w zasadzie dając Ci wszystko, co zrobił(a)byś w innym wypadku zgodnie z naszą instrukcją instalacji, lecz z poziomu interaktywnego instalatora.
+
+{{< caption-link url="https://marketplace.digitalocean.com/apps/mastodon" caption="Mastodon 1-Click Install Image na DigitalOceanie" >}}
```
Comment (hexandcube) [inline content/pl/user/run-your-own.md:51]:
```suggestion
{{< caption-link url="https://marketplace.digitalocean.com/apps/mastodon" caption="Mastodon 1-Click Install Image na DigitalOcean" >}}
```

File: content/pl/user/signup.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
+---
+title: Zarejestruj konto
```
Comment (hexandcube) [inline content/pl/user/signup.md:2]:
```suggestion
title: Zakładanie konta
```

File: content/pl/user/signup.md [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+
+Musisz wybrać stronę, na której zarejestrujesz się, tak jakbyś wybierał(-a) dostawcę poczty e-mail lub realm w World of Warcraft dla swojej nowej postaci. Ta strona będzie Twoim dostawcą usługi, hostem dla Twojego konta, Twoim profilem i Twoją stroną główną.
+
+{{< hint style="info" >}}
+Możesz [przeglądać listę wybranych serwerów według kategorii i języków na joinmastodon.org](https://joinmastodon.org/#getting-started).
+{{< /hint >}}
+
+### Rozumienie zasad strony {#tos}
+
+Zanim zarejestrujesz się, ważnym jest zapoznanie się z zasadami i regulaminem. Serwer Mastodona ma zwykle zapisane je na stronie `/about/more`, którą możesz znaleźć, klikając na „dowiedz się więcej” na stronie, którą widzisz, gdy nie jesteś zalogowany(-a).
```
Comment (hexandcube) [inline content/pl/user/signup.md:20]:
```suggestion
Zanim zarejestrujesz się, ważnym jest zapoznanie się z zasadami i regulaminem. Serwer Mastodona ma zwykle wypisane je na stronie `/about/more`, którą możesz znaleźć, klikając na „dowiedz się więcej” na stronie, którą widzisz, gdy nie jesteś zalogowany(-a).
```

File: content/pl/user/signup.md (hunk 18 above)
Comment (mkljczk) [reply content/pl/user/signup.md:2]: I feel the way I translated headlines/titles is just better for the ‘user guide' part of documentation, it feels more engaging to a user, although might be less accurate.

File: content/pl/user/contacts.md [hunk 20]
```diff
@@ -0,0 +1,45 @@
...
+
+{{< figure src="/assets/image%20%2849%29.png" caption="Wzajemni obserwujący, którzy nie przenieśli konta, sortowani według daty ostatniej aktywności" >}}
+
+W ustawieniach możesz znaleźć stronę zarządzania relacjami, pozwalającą na filtrowanie i sortowanie profili, z którymi jesteś połączony(-a), według różnych kryteriów:
+
+* **Relacje:** czy konto obserwuje Cię, jest obserwowane przez Ciebie, lub wzajemnie się obserwujecie.
+* **Stan konta:** czy konto jest obecnie oznaczone jako przekierowujące na inne.
+* **Aktywność konta:** czy konto publikowało jakieś wpisy w ciągu ostatniego miesiąca.
+
+Możesz wybrać które osoby chcesz przestać obserwować lub usunąć z obserwowanych, zaznaczając ich i klikając na odpowiedni przycisk w nagłówku tabeli.
```
[the remark this answers is not in the document]
Comment (mkljczk) [reply content/pl/user/contacts.md:26]: Dzięki, umknęłoby mi

File: content/pl/user/posting.md [hunk 21]
```diff
@@ -0,0 +1,134 @@
...
+
+{{< figure src="/assets/image%20%2826%29.png" caption="Edytuj media, aby dodać opis zawartości lub wybrać punkt skupienia dla miniaturki podglądu." >}}
+
+Po kliknięciu „Edytuj” na miniaturze załącznika, możesz załadować modal, który pozwoli na dodanie opisu mediów lub wybranie punktu skupienia. Choć jest to nieobowiązkowe, dobrze jest dodawać opis, który krótko opisze, co przedstawiane jest na mediach. Te opisy będą widoczne, gdy nie uda się załadować mediów z jakiegoś powodu, lub ktoś korzysta z czytnika zawartości ekranu, lub innej technologii wspomagającej. Ustawienie punktu skupienia jest również nieobowiązkowe, ale sprawi, że miniaturka wygląda lepiej, jeżeli zdjęcie nie ma wymiarów 16:9.
+
+### Ankiety {#polls}
+
+{{< figure src="/assets/image%20%2841%29.png" caption="Ankieta z dwoma opcjami, wygasająca po jednym dniu" >}}
+
+Naciśnij na ikonę wykresu, aby załączyć wykres do wpisu.
```
[the remark this answers is not in the document]
Comment (mkljczk) [reply content/pl/user/posting.md:65]: heh

File: content/pl/user/posting.md [hunk 22]
```diff
@@ -0,0 +1,134 @@
...
+* Ankiety są domyślnie jednokrotnego wyboru. Naciśnij na przycisk wyboru, aby przełączyć ją na ankietę wielokrotnego wyboru.
+* Ankiety mogą kończyć się po 5 minutach, 30 minutach, godzinie, 6 godzinach, dniu, 3 dniach lub tygodniu.
+
+## Poziomy publikacji {#privacy}
+
+| Poziom | Publiczna oś czasu | Odnośnik bezpośredni | Widok profilu | Domowe osi czasu |
+| :--- | :--- | :--- | :--- | :--- |
+| Publiczny | Tak | Tak | Tak | Tak |
+| Niewidoczny | Nie | Tak | Tak | Tak |
+| Tylko dla śledzących | Nie | Zalogowani na tym samym serwerze | W aplikacji lub zalogowani | Tak |
```
[the remark this answers is not in the document]
Comment (mkljczk) [reply content/pl/user/posting.md:77]: That's how it appears in the UI

File: content/pl/user/profile.md (hunk 10 above)
Comment (mkljczk) [reply content/pl/user/profile.md:57]: Actually, it uses *nazwa* | *zawartość* in the UI, so I'll use these terms

**Review by piotr-sikora-v — APPROVED** [review]
Very good translation
/Polish native

**Review by Gargron — APPROVED** [review]

**Pull request merged**